src/Encoder/Encoder.php

```diff
@@ -232,6 +218,16 @@ private static function isOnlyDoubleByteKanji(string $content) : bool
         return true;
     }
 
+    /**
+     * Checks if content only consists of alphanumeric characters (or is empty).
+     */
+    private static function isOnlyAlphanumeric(string $content) : bool
+    {
+        $allowed = '0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ $%*+-./:';
```

Comment:
Instead of assigning this variable on every call, would it make sense to have this as a private const on the class?